src/oas2/__tests__/accessors.test.ts

```diff
@@ -20,13 +20,90 @@ const securityDefinitionsFixture: Dictionary<Security> = {
 };
 
 describe('accessors', () => {
-  const securityFixture = [
+  const securityFixture: Array<Dictionary<string[], string>> = [
     {
       api_key: [],
       petstore_auth: ['write:pets', 'read:pets'],
     },
   ];
 
+  describe('relation between schemes', () => {
+    describe('when all of the given schemes are expected to be validated against', () => {
+      const securityFixtureWithAndRelation = securityFixture;
+
+      it('returns an array containing multiple elements', () => {
+        expect(
+          getSecurities(
+            {
+              securityDefinitions: securityDefinitionsFixture,
+              security: [],
+            },
+            securityFixtureWithAndRelation,
+          ),
+        ).toEqual([
+          [
+            {
+              in: 'header',
+              name: 'api_key',
+              type: 'apiKey',
+            },
+            {
```

Comment:
Is there any reason you can think of that we might need the security array's key here? Currently there is nothing tying this oauth2 object back to the security array. Does that make sense?

```suggestion
            {
              securityKey: 'petstore_auth',
```

Reply:
@lottamus, I think that given https://github.com/stoplightio/http-spec/pull/15/files#diff-f5ac86d0dd4d4fa1390aca28ac1fbc59L107-L112 it's OK to just not add this `securityKey`? If it was not a problem before, it shouldn't be a problem now. We can address it later if it turns out to be an issue